### DISCUSSION TITLE
Shop Packs: Collections inside Collections

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#354] [Suggestions]
For some ressources you can create a shop pack like 

Ingots
Stack 10    for XX Token
Stack 100  for XX Token
Stack 1000 For XX Token

But each Stack is an own sub package.. 

It would be great if we can create a shop package category and inside a shop package category for ingots as an example and then inside that, the available stacks..

Or (if I need to create a different suggestion) an amount field

Stack 10 buy 10 packs -> total 100 Ingots

**Comment by Decollates** [comment]
Yes, please. This would also be helpful for me where I have like a Chibi Collection for each tier of Ark Chibi. It'd be awesome if I could condense each Collection into a Mega Collection, or something.
